### DISCUSSION TITLE
Dark Theme - All Dropdown boxes white

### DISCUSSION THREAD

**Discussion by valdearg** [dani-garcia/vaultwarden#2253] [Q&A]
### Subject of the issue
When switching on the dark mode in the web vault, any dropdown box is plain white

![image](https://user-images.githubusercontent.com/2789033/150658007-dcc2d97f-dbce-4a5b-9a06-2a47ab8ecefd.png)

Tested in both Firefox and Chrome.

### Deployment environment

* vaultwarden version: 2.25.0

* Install method: docker image

* Clients used:  web vault in Firefox and Chrome

* Reverse proxy and version:  squid reverse proxy

* MySQL/MariaDB or PostgreSQL version: 

* Other relevant details:

### Steps to reproduce

Switched the Theme option to Dark (and Use Default Theme) via Settings > Options > Theme.

Select any of the dropdown boxes on this page, e.g. Vault Timeout, Language. Also happens on the Tools > Import data dropdown.

### Expected behaviour

Text should be visible, currently the text is very faint.

### Troubleshooting data

![image](https://user-images.githubusercontent.com/2789033/150658128-8df0f7e3-2ae1-45b5-ae80-6014f05c5a08.png)

**Comment by jjlin** [comment]
This is a known upstream issue: https://github.com/bitwarden/web/issues/1338

**Reply by MrSSvard** [reply]
It seems to have been fixed and released upstream: https://github.com/bitwarden/web/pull/1255

**Reply by BlackDex** [reply]
The same for Vaultwarden. The `testing` tagged images have the fix already for a while.

**Reply by 3XC1T3D** [reply]
> The same for Vaultwarden. The `testing` tagged images have the fix already for a while.

Hi @BlackDex ,

is there an ETA when the Commits from the testing Images will be merged into the latest Images?

**Comment by BlackDex** [comment]
Not exactly, but using testing would be the same. So i would suggest to use that right now.

**Reply by 3XC1T3D** [reply]
Actually i am using Vaultwarden for private use only. Is the testing image as stable as the latest image?

**Reply by BlackDex** [reply]
I'm using it my self. And there aren't any issues as far as i know currently.

Also, if the current main branch would be created into a new latest version, it will match testing 1:1.

**Reply by 3XC1T3D** [reply]
Thank you for reply. I think i will update to testing tomorrow 👍
